deployer/src/deployer/search/__init__.py

```diff
@@ -93,6 +93,8 @@ def get_progressbar():
         for success, info in streaming_bulk(
             connection,
             generator(),
+            max_chunk_bytes=52428800,  # default: 104857600
```

Comment:
Looks like we should also reduce the chunk size, see: https://github.com/mdn/dex/actions/runs/18172146864/job/51733316612#step:27:30

```
Counts - worked: 60,835 errors: 500
Most common errors....
500	TransportError(429, 'circuit_breaking_exception', '[parent] Data too large, data
```

```suggestion
            chunk_size=250,  # default: 500
            max_chunk_bytes=52428800,  # default: 104857600
```